tonic-build/src/client.rs

```diff
@@ -22,6 +22,7 @@ pub fn generate<T: Service>(service: &T, proto_path: &str) -> TokenStream {
             use tonic::codegen::*;
 
             #service_doc
+            #[derive(Debug)]
```

Comment:
We probably want to do a custom derive for this actually. We can hide the internals but have it output something like

```
Service { ... }
```